frontend/src/services/apiService.ts

```diff
@@ -13,14 +14,13 @@ const axiosInstance = axios.create({
 
 const apiRequest = (method: 'GET' | 'PUT' | 'POST' | 'DELETE') => async (
   url: string,
-  token?: string,
   data?: object
 ) =>
   axiosInstance
     .request({
       url,
       method,
-      headers: headers(token),
+      headers: headers(await authWrapper.authClient?.getTokenSilently()),
```

Comment:
Shame its only available like this :(

Still nice to not have to pass accessToken all through business/app logic :+1: